tests/e2e/scenarios/scalability/run-test.sh

```diff
@@ -53,30 +30,32 @@ if [[ -z "${K8S_VERSION:-}" ]]; then
   K8S_VERSION="$(curl -s -L https://dl.k8s.io/release/stable.txt)"
 fi
 
-# Download latest prebuilt kOps
-if [[ -z "${KOPS_BASE_URL:-}" ]]; then
-  KOPS_BASE_URL="$(curl -s https://storage.googleapis.com/kops-ci/bin/latest-ci.txt)"
-fi
-export KOPS_BASE_URL
+## FETCH MAHAMED's perf-tests fork
+git -C "${GOPATH}"/src/k8s.io/perf-tests remote add dev-fork https://github.com/borg-land/perf-tests.git
```

Comment:
I think this `borg-land fork repo`  was used for testing purposes and forgot to change it back ?

Reply:
Removed